docs/legacy-project-tips.md

```diff
@@ -63,6 +63,44 @@ linting process more manageable, we're going to provide a
 some tips of tricks to break things down into bite size
 chunks.
 
+Pre-commit hook to lint only changed files
+------------------------------------------
+
+Using npm-scripts, some bash-fu and the [husky][husky] 
+module, you can have lint-trap only lint changed *.js files
+as a pre-commit hook. To do this, first install husky as a
+dev dependency:
+
+    $ npm install --save-dev husky
+
+Then add the following to your `package.json` file:
+
+    "scripts": {
+        ...
+        "lint-changed": "lint-trap `git diff HEAD --name-only --diff-filter=ACMRTU | awk '/\.js$/' | awk  -v path=$(git rev-parse --show-toplevel)/ '{print path$1}'`",
+        "precommit": "npm run lint-changed",
+        ...
+    },
+
+Husky will install git-hooks, including a pre-commit hook
```

Comment:
because we use it for everything else. Author has taken several of my patches and is respectful.